clone/src/lib.rs

```diff
@@ -77,39 +87,63 @@ impl Args for Options {
-        seed: options.seed.clone(),
-        verbose: false,
-        force: false,
-    })?;
-
-    // Tracking influences the checkout (by creating additional remotes),
-    // so we run it first.
-    if options.track {
-        let profile = profile::default()?;
-        let sock = keys::ssh_auth_sock();
-        let (_, storage) = keys::storage(&profile, sock)?;
-        let cfg = tracking::config::Config::default();
-
-        tracking::track(
-            &storage,
-            &options.urn,
-            None,
-            cfg,
-            tracking::policy::Track::Any,
-        )??;
-    }
-    let path = rad_checkout::execute(rad_checkout::Options { urn: options.urn })?;
-
-    if let Some(seed_url) = options.seed.seed_url() {
-        seed::set_seed(&seed_url, seed::Scope::Local(&path))?;
-        term::success!(
-            "Local repository seed for {} set to {}",
-            term::format::highlight(path.display()),
-            term::format::highlight(seed_url)
-        );
+    match options.urn_url {
+        UrnUrl::Urn(urn) => {
+            rad_sync::run(rad_sync::Options {
+                fetch: true,
+                urn: Some(urn.clone()),
+                seed: options.seed.clone(),
+                verbose: false,
+                force: false,
+            })?;
+
+            // Tracking influences the checkout (by creating additional remotes),
+            // so we run it first.
+            if options.track {
+                let profile = profile::default()?;
+                let sock = keys::ssh_auth_sock();
+                let (_, storage) = keys::storage(&profile, sock)?;
+                let cfg = tracking::config::Config::default();
+
+                tracking::track(&storage, &urn, None, cfg, tracking::policy::Track::Any)??;
+            }
+
+            let path = rad_checkout::execute(rad_checkout::Options { urn })?;
+
+            if let Some(seed_url) = options.seed.seed_url() {
+                seed::set_seed(&seed_url, seed::Scope::Local(&path))?;
+                term::success!(
+                    "Local repository seed for {} set to {}",
+                    term::format::highlight(path.display()),
+                    term::format::highlight(seed_url)
+                );
+            }
+        }
+        UrnUrl::Url(url) => {
+            let proj = url
+                .path_segments()
+                .ok_or(anyhow!("Couldn't get segments of URL"))?
+                .last()
+                .ok_or(anyhow!("Couldn't get last segment of URL"))?;
+            let proj = proj.strip_suffix(".git").or(Some(proj)).unwrap();
+
+            let spinner = term::spinner(&format!("Cloning {} locally", term::format::bold(&proj)));
+            let status = Command::new("git")
+                .arg("clone")
+                .arg(url.as_str())
+                .stdout(Stdio::null())
+                .stderr(Stdio::null())
+                .status()
+                .context("Failed to call 'git clone'")?;
```

Comment:
We have a helper for this, `git::git()`

Reply:
added it here https://github.com/radicle-dev/radicle-client-tools/pull/46/files#diff-fdb6901b1fe055e8ed5750ff9dac0ee88b6a703de14b4722443ea6e9cc695adaR306-R309